### DISCUSSION TITLE
Merge Latest MelonDs Update 0.9.3

### DISCUSSION THREAD

**Discussion by Bittu2002** [rafaelvcaetano/melonDS-android#620] [Ideas]
Last time you Merge latest melonDS changes in Beta 1.4.0 that released on Mar 29 ! please update that . As well as update JIT And internal libs so performance increase :D

**Comment by oliwek** [comment]
0.9.3 added a DSiWare title manager, that would be awesome on Android !

**Reply by rafaelvcaetano** [reply]
It will be added :) I'll try to include it for the next release. I'm just not quite sure what is the best UX and UI for that.

**Comment by Haxorzz** [comment]
0.94 is here!

**Reply by Bardock88** [reply]
> 0.94 is here!

No. Is v0.9.4.
Will there be anything new in relation to this version for android?
